main.lua

```diff
@@ -75,67 +79,76 @@ function ComicMeta:onComicMeta()
                 handle:close()
             end
 
-            if xml_content and #xml_content > 0 then
-                -- Parse the XML content and create a metadata table
-                local metadata = {
-                    title = xml_content:match("<Title>(.-)</Title>"),
-                    authors = xml_content:match("<Writer>(.-)</Writer>"),
-                    series = xml_content:match("<Series>(.-)</Series>"),
-                    series_index = xml_content:match("<Number>(.-)</Number>"),
-                    description = xml_content:match("<Summary>(.-)</Summary>"),
-                }
-
-                -- Fixup metadata
-                for key, value in pairs(metadata) do
-                    if key == "title" then
-                        metadata[key] = util.htmlEntitiesToUtf8(value)
-                    elseif key == "authors" then
-                            metadata[key] = util.htmlEntitiesToUtf8(value)
-                    elseif key == "series" then
-                        metadata[key] = util.htmlEntitiesToUtf8(value)
-                    elseif key == "series_index" then
-                        metadata[key] = tonumber(util.htmlEntitiesToUtf8(value))
-                    elseif key == "description" then
-                        -- Description may (often in EPUB, but not always) or may not (rarely in PDF) be HTML
-                        metadata[key] = util.htmlToPlainTextIfHtml(util.htmlEntitiesToUtf8(value))
-                    end
-                end
+            if not xml_content or #xml_content == 0 then
+                return
```

Comment:
oooh wait, we're in a loop, so this should be a continue but Lua has no continue xD.
So we actually do need the indentation level. 
I'm in agonizing despair to get reminded that Lua has no continue, I'll revert the indentation improvement 🙈 

I think this can be improved by breaking down the code in multiple smaller functions and then the indentation becomes a bit easier to follow!